src/Agent/NewRelic/Agent/Core/Metrics/MetricNames.cs

```diff
@@ -840,6 +840,8 @@ public static string GetSupportabilityInstallType(string installType)
         public const string SupportabilityGCSamplerV2Enabled = SupportabilityDotnetPs + "GCSamplerV2/Enabled";
         public const string SupportabilityAwsAccountIdProvided = SupportabilityDotnetPs + "AwsAccountId/Config";
         public const string SupportabilityAzureFunctionModeEnabled = SupportabilityDotnetPs + "AzureFunctionMode/Enabled";
+        public const string SupportabilityAspNetCore6PlusBrowserInjectionEnabled = SupportabilityDotnetPs + "AspNetCore6PlusBrowserInjection/Enabled";
```

Comment:
I know we haven't been consistent, but other metrics use the  constants `Enabled` and `Disabled` (which have the values `enabled` and `disabled` respectively) instead of a string literal. 

There's also a pattern for calling a method and passing a bool to get the metric name -- see `SupportabilityInfiniteTracingCompression(bool compressionEnabled)` for example.

My Azure Function in-process PR has adopted that pattern; you might consider using it here as well.